src/swap/SwapReviewScreen.test.tsx

```diff
@@ -152,25 +154,80 @@ describe('SwapReviewScreen', () => {
       // Swap To
       expect(getByTestId('ToSwapAmountToken')).toHaveTextContent('3.10 cUSD')
       // Exchange Rate
-      expect(getByText('1 CELO ≈ 3.10 cUSD')).toBeTruthy()
+      expect(getByTestId('ExchangeRate')).toHaveTextContent('1 CELO ≈ 3.10 cUSD')
       // Estimated Gas
       expect(getByTestId('EstimatedGas')).toHaveTextContent('0.00015 CELO')
       // Swap Fee
       expect(getByTestId('SwapFee')).toHaveTextContent('swapReviewScreen.free')
     })
   })
 
+  it('should display correct exchange rate when buyAmount is used', async () => {
+    const newStore = {
+      ...store,
+      swap: {
+        ...store.swap,
+        swapUserInput: {
+          ...store.swap.swapUserInput,
+          toToken: mockCeloAddress,
+          fromToken: mockCusdAddress,
+          swapAmount: {
+            FROM: '3100000000000000000',
+            TO: '1000000000000000000',
+          },
+          updatedField: Field.TO,
+        },
+      },
+    }
+
+    const newMockStore = createMockStore(newStore)
+
+    mockFetch.mockResponse(
+      JSON.stringify({
+        unvalidatedSwapTransaction: {
+          sellToken: mockCusdAddress,
+          buyToken: mockCeloAddress,
+          buyAmount: '1000000000000000000',
```

Comment:
I think we should use a different amount here to make sure the price is showing as a unit.
And is not inadvertently reusing the buyAmount.
Same for the other examples.

Reply:
Adjusted the values for the `should display correct exchange rate when buyAmount is used` test in https://github.com/valora-inc/wallet/pull/3291/commits/10fa405b099e6b47aa87169f751558a1f03b0b05!